test/spec/clinic/dashboardController.spec.js

```diff
@@ -58,6 +65,7 @@ describe('DashboardController', function () {
     it('should get patient uuid', function () {
       $rootScope.$apply();
       expect(patientService.getPatient).toHaveBeenCalled();
+      expect(alertService.get).toHaveBeenCalled();
```

Comment:
I would recommend that instead of checking if the method was called we check if $scope.flags has the right data, preferably not empty, because that's what ultimately makes the alerts show up on the screen.